test/e2e/testsuite/testsuite.go

```diff
@@ -299,6 +299,14 @@ func InitializeScenario(s *godog.ScenarioContext) {
 					fmt.Println(err)
 					os.Exit(1)
 				}
+
+				if runtime.GOOS == "linux" {
+					err = crcCmd.UnsetConfigPropertySucceedsOrFails("network-mode", "succeeds") // unsetting property that is not set gives 0 exitcode, so this works
+					if err != nil {
+						fmt.Println(err)
+						os.Exit(1)
+					}
+				}
```

Comment:
you really doesn't need to use platform (linux) condition here because when you unset the network-mode property it would go to default which is for linux => system-mode and for other platform user mode.

So only following should work.

```golang
err = crcCmd.UnsetConfigPropertySucceedsOrFails("network-mode", "succeeds") // unsetting property that is not set gives 0 exitcode, so this works
					if err != nil {
						fmt.Println(err)
						os.Exit(1)
					}
```

Reply:
@praveenkumar I didn't add the platform condition at the beginning, but the check on Windows failed.

Reply:
The unset command will fail on both windows and mac platform. 
Check in windows:
```
PS C:\Users\crcqe> crc config unset network-mode
Configuration property 'network-mode' does not exist
PS C:\Users\crcqe> echo $?
False
```

Check in mac:
```
lul@lul-mac hosts % crc config unset network-mode
Configuration property 'network-mode' does not exist
lul@lul-mac hosts % echo $?
1
```

Reply:
Ah `network-mode` is added for linux only :(